src/Game.tsx

```diff
@@ -7,7 +7,7 @@ import { Movie } from "./Movie-Interface";
 import MysteryMovie from "./MysteryMovie";
 
 export default function Game() {
-    const [currentGuess, setCurrentGuess] = useState<Movie>({title: "Guessed Movie", year: "", gross: "", director: "", distributor: "", genres: [], actors: [], tagline: "", plot: "", poster: "",})
+    const [currentGuess, setCurrentGuess] = useState<Movie>({id: NaN, title: "Guessed Movie", year: "", gross: "", director: "", distributor: "", genres: [], actors: [], tagline: "", plot: "", poster: "",})
```

Comment:
we can define an initial state variable outside of the component & use that here so it's less cluttered.